core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -84,6 +90,89 @@ public void register(BuiltinFunctionRepository repository) {
     repository.register(to_days());
     repository.register(week());
     repository.register(year());
+
+    repository.register(now());
+    repository.register(current_timestamp());
+    repository.register(localtimestamp());
+    repository.register(localtime());
+    repository.register(sysdate());
+    repository.register(curtime());
+    repository.register(current_time());
+    repository.register(curdate());
+    repository.register(current_date());
+  }
+
+  /**
+   * NOW() returns a constant time that indicates the time at which the statement began to execute.
+   */
+  private LocalDateTime now(@Nullable Integer fsp) {
+    return formatLocalDateTime(QueryContext::getProcessingStartedTime, fsp);
```

Comment:
I think queryContext is not proper way to store startedTime. My concern is contextContext is init for each rest request, but in future, if we plan to support scheduled query execution, which required execute query at background. 
My suggestions is rewrite now as literal value in AstBuilder. which means
1. Now is only executed one time.
2. SysDate is executed multiple times.

Reply:
Please, see rework in https://github.com/Bit-Quill/opensearch-project-sql/pull/113

Reply:
looks good.

Reply:
Cool! All these changes are here, please re-review.

cc: @penghuo @dai-chen